CHANGELOG.md

```diff
@@ -4,6 +4,32 @@
 
 **Language**:
 
+- _Breaking:_ Case syntax now uses brackets `[]` rather than braces `{}`.
+
+  To convert previous PRQL queries to this new syntax simply change
+  `case { ... }` to `case [ ... ]`.
+
+**Features**:
+
+**Fixes**:
+
+- Update case syntax to brackets `case [ ... ]` over braces `case { ... }`
+  (@AaronMoat, #3517)
```

Comment:
I think we can consolidate this into the changelog above, so we only have one entry?